src/user-event/__tests__/type-modifiers.js

```diff
@@ -24,6 +24,7 @@ test('{esc} triggers typing the escape character', async () => {
     Events fired on: input[value=""]
 
     focus
+    select
```

Comment:
All of the `select` events are coming from calling `setSelectionRange` on the input...

Not sure how to avoid this but I think we should try... Will circle back on this when we do the audit.